### PR TITLE
Adds the `hideHeaders` prop

### PR DESCRIPTION
**Pull request by CharlesOkwuagwu**

### CODE REVIEW

File: src/components/VDataTable/mixins/head.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 export default {
   methods: {
     genTHead () {
+      if (this.hideHeaders === true) { return } // Exit Early since no headers are needed. Do we also need to set this.headers = [] ?
```
Comment (KaelWD) [inline src/components/VDataTable/mixins/head.js]: Don't ask questions like that in the code, put them in a message in the PR instead.

Comment (jacekkarczmarczyk) [reply src/components/VDataTable/mixins/head.js]: And since `hideHeaders` is bool, `=== true` can be removed

Comment (KaelWD) [reply src/components/VDataTable/mixins/head.js]: *And* you don't need the braces either, all they do is look ugly

Comment (CharlesOkwuagwu) [reply src/components/VDataTable/mixins/head.js]: All noted

Comment (KaelWD) [reply src/components/VDataTable/mixins/head.js]: And to answer the question, you can't set headers, as it is a prop.

Comment (CharlesOkwuagwu) [reply src/components/VDataTable/mixins/head.js]: OK. Got it

**Comment by johnleider** [comment]
Add a PR on the docs for the new field

**Comment by CharlesOkwuagwu** [comment]
Done. Do we need an example?

**Comment by johnleider** [comment]
Yes

**Comment by johnleider** [comment]
Will merge with 1.0 release

**Pull request merged**